packages/paste-core/components/disclosure/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@twilio-paste/disclosure",
+  "version": "0.0.0",
+  "category": "interaction",
+  "status": "alpha",
+  "description": "The Disclosure is used to create accessible, hierarchical and collapsible structure to your pages.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:dev": "yarn clean && yarn compile:dev",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.build.tsbuildinfo && rm -rf .rpt2_cache",
+    "compile": "rollup -c --environment NODE_ENV:production",
+    "compile:dev": "rollup -c --environment NODE_ENV:development",
+    "prepublishOnly": "yarn build",
+    "type-check": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "@twilio-paste/absolute": "^2.0.46",
```

Comment:
I think we can drop Absolute from here, and the devDeps below.

Reply:
Yup, good catch. I’m no longer using Button so I can remove a bunch of these now

Reply:
I think we can remove paste button and spinner too